pages/common/zip2john.md

```diff
@@ -0,0 +1,16 @@
+# zip2john
+
+> A tool to extract password hashes from zip files for use with John the Ripper password cracker.
+> This is a utility tool usually installed as part of the John the Ripper installation.
```

Comment:
please add a link to this page

Reply:
```suggestion
> This is a utility tool usually installed as part of the John the Ripper installation.
> More information: <https://github.com/openwall/john>.
```
I found out that this tool is present under `john`. References [1](https://www.openwall.com/john/), [2](https://secnhack.in/crack-zip-files-password-using-john-the-ripper/).

Or we can use 
```suggestion
> This is a utility tool usually installed as part of the John the Ripper installation.
> More information: <https://www.openwall.com/john/>.
```
to align with the other page.
https://github.com/tldr-pages/tldr/blob/d144d74b35cd702763c1883c100f620c43ad8aa1/pages/common/john.md?plain=1#L4